generate_workspace/src/main/java/com/google/devtools/build/workspace/output/BzlWriter.java

```diff
@@ -41,7 +43,9 @@ public BzlWriter(String[] argv, String outputDirStr) {
 
   @Override
   public void write(Collection<Rule> rules) {
-    try (PrintStream outputStream = new PrintStream(generatedFile.toFile())) {
+    try {
```

Comment:
It's going to throw an IOException as well. You would prefer two separate try-catch blocks to one?